app/assets/stylesheets/application.css

```diff
@@ -14,7 +14,7 @@
  *= require_self
  */
  body {
-    background: black;
+    
```

Comment:
If you don't occupy this style (body), please delete it.

Reply:
done.